components/Taxation/basic_usage.rst

```diff
@@ -0,0 +1,179 @@
+    use Sylius\Component\Taxation\Model\TaxCategory;
+
+    $taxRate = new TaxRate();
+    $taxCategory = new TaxCategory();
+
+    $taxRate->setAmount(0.5);
+    $taxRate->getAmount(); // Output will be 0.5
+    $taxRate->getAmountAsPercentage(); // Output will be 50
+
+
+Setting Tax Category
+~~~~~~~~~~~~~~~~~~~~
+
+Every tax rate can have a tax category. You can simply set or unset it by calling ``->setCategory()``.
+
+.. code-block:: php
+
+    <?php
+
+    $taxRate->setCategory($taxCategory);
+    $taxRate->getCategory(); // Output will be  $taxCategory object
+    $taxRate->setCategory();
+    $taxRate->getCategory(); // Output will be null
+
+
+Including tax rate in price
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also included a tax rate in price by calling ``setIncludedInPrice(true)`` (false by default).
+To check if tax rate is included in price call ``isIncludedInPrice()``.
+
+.. hint::
+    You can read how this property influences on price in chapter :ref:`default-calculator`.
+
+
+Setting calculator
+~~~~~~~~~~~~~~~~~~
+
+To set type of calculator for your tax rate object call ``setCalculator('nameOfCalculator')``. Notice that nameOfCalculator
+should be the same as name of your calculator object.
+
+.. hint::
+    To understand meaning of this property go to :ref:`delegating-calculator`.
+
+
+Tax Category
+------------
+
+Every tax category  has two identifiers, an ID and name. You can access those by calling ``->getId()`` and ``getName()``
+respectively. The name is mutable, so you can change it by calling ``->setName('Clothing')`` on the tax category instance.
+
+Tax Rate Management
+~~~~~~~~~~~~~~~~~~~
+The collection of tax rates (Implementing the ``Doctrine\Common\Collections\Collection`` interface) can be obtained using
+the ``getRates()`` method. To add or remove tax rates, you can use the ``addRate()`` and ``removeRate()`` methods.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Taxation\Model\TaxRate;
+    use Sylius\Component\Taxation\Model\TaxCategory;
+
+    $taxCategory = new TaxCategory();
+
+    $taxRate1 = new TaxRate();
+    $taxRate1->setName('taxRate1');
+
+    $taxRate2 = new TaxRate();
+    $taxRate2->setName('taxRate2');
+
+    $taxCategory->addRate($taxRate1);
+    $taxCategory->addRate($taxRate2);
+    $taxCategory->getRates();
+    //returns a collection of objects that implement the TaxRateInterface
+    $taxCategory->removeRate($taxRate1);
+    $taxCategory->hasRate($taxRate2); // returns true
+    $taxCategory->getRates(); // returns collection with one element
+
+Calculators
+-----------
+
+.. _default-calculator:
+
+Default Calculator
+~~~~~~~~~~~~~~~~~~
+
+**Default Calculator** gives you the ability to calculate the tax amount for given base amount and tax rate.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Taxation\Model\TaxRate;
+    use Sylius\Component\Taxation\Calculator\DefaultCalculator;
+
+    $taxRate = new TaxRate();
+    $taxRate->setAmount(2);
```

Comment:
This tax rate is 200%, which does not sound realistic. Paying 20$ tax for a 10$ purchase, that's a bit crazy. :D :penguin: